src/components/bulk-email-tool/bulk-email-task-manager/BulkEmailPendingTasks.jsx

```diff
@@ -1,32 +1,89 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import { useParams } from 'react-router-dom';
 
-import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { DataTable } from '@edx/paragon';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { getInstructorTasks } from './api';
+import messages from './messages';
+import useInterval from '../../../utils/useInterval';
 
-export default function BulkEmailPendingTasks() {
+export function BulkEmailPendingTasks({ intl }) {
   const { courseId } = useParams();
 
-  const [instructorTaskData, setInstructorTaskData] = useState(); // eslint-disable-line no-unused-vars
+  const [instructorTaskData, setInstructorTaskData] = useState();
 
-  useEffect(() => {
+  useInterval(() => {
     async function fetchPendingInstructorTasksData() {
       const data = await getInstructorTasks(courseId);
       const { tasks } = data;
       setInstructorTaskData(tasks);
     }
-    fetchPendingInstructorTasksData();
-  }, []);
+    try {
+      fetchPendingInstructorTasksData();
+    } catch (error) {
+      // do nothing
```

Comment:
I didn't want to just console log the error here but couldn't think of anything else we would want to do. If we can't fetch/refresh the data then we don't want to do reset it or anything, I think we just keep whatever our current results are and keep the table state as is.

I'd be swallowing the error here but I don't want the page to crash if we can't get results back from the API for whatever reason.

Reply:
You could use the paragon `Form.Control.Feedback` component for this and create a generic error message to give feedback to the user that something failed. If you use it together with `Form.Group` you shouldn't need any additional styling: https://paragon-edx.netlify.app/components/form/form-group/